tests/postprocessing/test_sqlite_results.py

```diff
@@ -0,0 +1,344 @@
+"""Tests for SQLite simulation results reader."""
+
+import numpy as np
+import pytest
+
+from movici_simulation_core.postprocessing.sqlite_results import (
+    SQLiteSimulationResults,
+    detect_results_format,
+    get_simulation_results,
+)
+from movici_simulation_core.storage.sqlite_schema import SimulationDatabase
+
+# Skip tests if sqlalchemy not available
+pytest.importorskip("sqlalchemy")
+
```

Comment:
Make sure that sqlalchemy is available in ci, so that we don't accidentally skip these tests